package.json

```diff
@@ -104,7 +104,7 @@
     "rimraf": "^2.6.1",
     "shelljs": "^0.8.2",
     "stylelint": "^9.2.0",
-    "stylelint-config-terra": "^1.0.0",
+    "stylelint-config-terra": "git+https://github.com/cerner/stylelint-config-terra.git#Errors",
```

Comment:
We should update this similar to terra-core to avoid the git dependency until stylelint is released with a major version. 

https://github.com/cerner/terra-core/blob/master/package.json#L27-L36

Reply:
Fixed. https://github.com/cerner/terra-framework/pull/429/commits/0f375313c8f4c80a343c560d05a3fb077f163bd9